src/Process/Deploy/InstallUpdate/ConfigUpdate/SearchEngine/Config.php

```diff
@@ -124,7 +124,10 @@ private function getElasticSearchConfiguration(array $config)
     {
         $engine = 'elasticsearch';
 
-        if (Semver::satisfies($this->elasticSearch->getVersion(), '>= 5')) {
+        $esVersion = $this->elasticSearch->getVersion();
+        if (Semver::satisfies($esVersion, '>= 6')) {
```

Comment:
just for the future. If tendency is the same, we just need to retrieve major version for engine name which is higher v.5

Reply:
Nice catch, changed to this logic